packages/application-shell/src/components/custom-view-dev-host/custom-view-dev-host.tsx

```diff
@@ -0,0 +1,146 @@
+import { ReactNode, useState } from 'react';
+import { Route, Router, Switch } from 'react-router-dom';
+import { CustomViewLoader } from '@commercetools-frontend/application-components';
+import { type TApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import history from '@commercetools-frontend/browser-history';
+import { type TAsyncLocaleDataProps } from '@commercetools-frontend/i18n';
+import Constraints from '@commercetools-uikit/constraints';
+import PrimaryButton from '@commercetools-uikit/primary-button';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import {
+  TCustomView,
+  TCustomViewSize,
+  TCustomViewStatus,
+  TCustomViewType,
+} from '../../types/generated/settings';
+import ApplicationShell from '../application-shell';
+
+const defaultCustomViewConfig: TCustomView = {
+  createdAt: '',
+  defaultLabel: '',
+  id: Date.now().toString(),
+  installedBy: [],
+  labelAllLocales: {},
+  locators: [],
+  owner: {
+    createdAt: '',
+    id: '',
+    organizationId: '',
+    updatedAt: '',
+  },
+  ownerId: '',
+  permissions: [],
+  status: TCustomViewStatus.Draft,
+  type: TCustomViewType.CustomPanel,
+  typeSettings: {
+    size: TCustomViewSize.Small,
+  },
+  updatedAt: '',
+  url: '',
+};
```

Comment:
Does this need to be hardcoded here? Isn't this parsed and loaded from the config file? 🤔

Reply:
We don't have all the information in the config file.
This will come from the host application in runtime but I didn't want to have a null value in the context for starters.